### DISCUSSION TITLE
Adding Custom Marker Icons

### DISCUSSION THREAD

**Discussion by VictordotCC** [gmplot/gmplot#191] [Q&A]
Hi!

Is there any way to add or to choose a custom Marker Icon when adding it to a map?

**Comment by frslm** [comment]
I haven't tried this myself, but you could try overwriting `_COLOR_ICON_PATH` right after importing `gmplot`, so something like:
```python
import gmplot

_COLOR_ICON_PATH = "www.website.com/%s.png" # (or local path: "/custom_markers/%.png")

... # (code to plot markers)
```

**Reply by Tanaka86** [reply]
Hello @frslm , can you explain then how do you use _COLOR_ICON_PATH ?
If I have in my project  my_icon.png 

```
import gmplot
_COLOR_ICON_PATH =   "/my_icon/%.png")
....
gmap.marker(long, lat, info_window)
outfile = 'myprojet.html'
gmap.draw(outfile)
```

Thanks

**Reply by Tanaka86** [reply]
Hi @frslm 
I tried to overwrite `_COLOR_ICON_PATH` but it doesn't work. Do you have any idea to make it ?
